okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -293,7 +294,7 @@ public void close() throws SecurityException {
     Buffer buffer = createMessageFrame(message);
     frameHandler().data(false, 3, buffer, (int) buffer.size(),
         (int) buffer.size());
-    assertThat(logs).hasSize(1);
+    assertWithMessage("log messages: " + logs).that(logs).hasSize(1);
```

Comment:
How does this help? I tried it out and it seems to change:
```
io.grpc.okhttp.OkHttpClientTransportTest > testClientHandlerFrameLogger FAILED
    value of    : iterable.size()
    expected    : 2
    but was     : 1
    iterable was: [java.util.logging.LogRecord@7f12d356]
```

into:
```
io.grpc.okhttp.OkHttpClientTransportTest > testClientHandlerFrameLogger FAILED
    log messages: [java.util.logging.LogRecord@45026a28]
    value of    : iterable.size()
    expected    : 2
    but was     : 1
    iterable was: [java.util.logging.LogRecord@45026a28]
```

The point of Truth is that it prints the value of what your testing. Why print it twice?